regression/cbmc-java/lambda2/test_no_crash.desc

```diff
@@ -0,0 +1,6 @@
+CORE
+SymStream.class
+--verbosity 10 --show-goto-functions
```

Comment:
Perhaps add `EXIT=0` to ensure it really doesn't crash!

Reply:
+1, and add `SIGNAL` - cf. #1905.

Reply:
I completed the regression tests with tests on `EXIT` and `SIGNAL`